ecosystem/sep-0010.md

```diff
@@ -18,57 +18,58 @@ This SEP defines the standard way for clients such as wallets or exchanges to cr
-1. The client verifies that the transaction has an invalid sequence number 0.  This is extremely important to ensure the transaction isn't malicious.
-1. The client verifies that the transaction is signed by the `SIGNING_KEY` obtained through discovery flow.
-1. The client verifies that the transaction's first operation is a Manage Data operation that has its:
-   1. Source account set to the user's account.
-   1. Key set to `<home domain> auth` where the home domain is the home domain from the discovery flow.
+
+1. The **Client** obtains a unique [`challenge`](#challenge) from the **Server**, which is represented as specially formed Stellar transaction
+1. The **Client** verifies that the transaction has an invalid sequence number 0.  This is extremely important to ensure the transaction isn't malicious.
+1. The **Client** verifies that the transaction is signed by the **Server Account** obtained through discovery flow.
+1. The **Client** verifies that the transaction's first operation is a Manage Data operation that has its:
+   1. Source account set to the **Client Account**
+   1. Key set to `<home domain> auth` where the home domain is the **Home Domain**.
    1. Value set to a nonce value.
-1. The client verifies that if the transaction has a Manage Data operation with key `web_auth_domain` that it has:
-   1. Source account set to the server's account.
-   1. Key set to `web_auth_domain`.
-   1. Value set to the domain name of the SEP-10 server from which the client requested the challenge.
-1. The client verifies that if the transaction has other operations they are Manage Data operations that all have their source accounts set to the the server's account.
-1. The client signs the transaction using the secret key(s) of signers for the user's Stellar account
-1. The client submits the signed challenge back to the server using [`token`](#token) endpoint
-1. The server checks that the user's account exists
-1. If the user's account exists:
-  1. The server gets the signers of the user's account
-  1. The server verifies the client signatures count is one or more;
-  1. The server verifies the client signatures on the transaction are signers of the user's account;
-  1. The server verifies the weight provided by the signers meets the required threshold(s), if any
-  1. If the signatures check out, the server responds with a [JWT](https://jwt.io) that represents the user's session
-1. If the user's account does not exist (optional):
-  1. The server verifies the client signature count is one
-  1. The server verifies the client signature is correct for the master key of the account
-1. Any future calls to the server can be authenticated by including the JWT as a parameter
+1. The **Client** verifies that if the transaction has a Manage Data operation with key `web_auth_domain` that it has:
+   1. Source account set to the **Server Account**.
+   1. Value set to the **Server**'s domain.
```

Comment:
❗ I think we need to keep the text that was removed. It is important that the client verify it matches the domain they requested from. I don't want that distinction to be lost.
```suggestion
   1. Value set to the **Server**'s domain that the client requested the challenge from.
```